packages/app/src/components/extension-button.tsx

```diff
@@ -0,0 +1,93 @@
+import React, { FC } from 'react';
+import { Box, Text, ChromeIcon, Flex, PseudoBox, FirefoxIcon, ChevronIcon } from '@blockstack/ui';
+import { PoweredBy } from '@blockstack/connect';
+import { buildEnterKeyEvent } from '@blockstack/connect/react/components/link';
+
+// https://stackoverflow.com/a/13348618
+const isChrome = () => {
+  const isChromium = window.chrome;
+  const winNav = window.navigator;
+  const vendorName = winNav.vendor;
+  const isOpera = typeof (window as any).opr !== 'undefined';
+  const isIEedge = winNav.userAgent.includes('Edge');
+  const isIOSChrome = /CriOS/.exec(winNav.userAgent);
+
+  if (isIOSChrome) {
+    return false;
+  } else if (
+    isChromium !== null &&
+    typeof isChromium !== 'undefined' &&
+    vendorName === 'Google Inc.' &&
+    isOpera === false &&
+    isIEedge === false
+  ) {
+    return true;
+  } else {
+    return false;
+  }
+};
+
+interface Browser {
+  name: 'Chrome' | 'Firefox';
+  icon: typeof FirefoxIcon | typeof ChromeIcon;
+}
```

Comment:
Do we need 'other'?

Reply:
Nope - if the code detects that it's not Chrome or Firefox, then it simply shows the existing "Powered By Blockstack" footer.